### PR TITLE
fixes bug 1237703 - Django 1.8

### PR DESCRIPTION
**Pull request by peterbe**

### CODE REVIEW

File: webapp-django/crashstats/authentication/management/commands/makesuperuser.py [hunk 1]
```diff
@@ -10,7 +10,8 @@ def get_input(text):
 
 class Command(BaseCommand):
 
-    help = 'emailaddress [, otheremailaddress, ...]'
+    def add_arguments(self, parser):
+        parser.add_argument('emailaddress', nargs='+', type=str)
```
Comment (adngdb) [inline webapp-django/crashstats/authentication/management/commands/makesuperuser.py]: So you require the argument to be passed with the command, but then it still asks you to write it a second time.

File: webapp-django/crashstats/settings/base.py [hunk 2]
```diff
@@ -42,15 +53,9 @@ def path(*dirs):
...
-    'django.template.loaders.filesystem.Loader',
-    'django.template.loaders.app_directories.Loader',
-)
-
-TEMPLATE_DIRS = (
-    path('templates'),
-)
+# TEMPLATE_DIRS = (
+#     path('templates'),
+# )
```
Comment (adngdb) [inline webapp-django/crashstats/settings/base.py]: Why keep this commented out?

File: webapp-django/crashstats/settings/base.py [hunk 3]
```diff
@@ -160,15 +209,16 @@ def JINJA_CONFIG():
...
-}
+# # This disables all mail_admins on all django.request errors.
+# # We can do this because we use Sentry now instead
+# LOGGING = {
+#     'loggers': {
+#         'django.request': {
+#             'handlers': []
+#         }
+#     }
+# }
```
Comment (adngdb) [inline webapp-django/crashstats/settings/base.py]: And same here.

File: webapp-django/crashstats/settings/base.py [hunk 4]
```diff
@@ -114,9 +120,9 @@ def JINJA_CONFIG():
 
 # Because Jinja2 is the default template loader, add any non-Jinja templated
 # apps here:
-JINGO_EXCLUDE_APPS = (
-    'browserid',
-)
+# JINGO_EXCLUDE_APPS = (
+#     'browserid',
+# )
```
Comment (adngdb) [inline webapp-django/crashstats/settings/base.py]: Same here.

**Comment by adngdb** [comment]
A few nits but nothing seemed to be broken during my local tests. r+

File: webapp-django/crashstats/authentication/management/commands/makesuperuser.py (hunk 1 above)
Comment (peterbe) [inline webapp-django/crashstats/authentication/management/commands/makesuperuser.py]: What do you mean?

File: webapp-django/crashstats/authentication/management/commands/makesuperuser.py [hunk 5]
```diff
@@ -10,7 +10,8 @@ def get_input(text):
...
 
     def handle(self, *args, **options):
         if not args:
```
Comment (willkg) [inline webapp-django/crashstats/authentication/management/commands/makesuperuser.py]: I think this check is wrong. If you're using the `add_arguments` thing, then it uses argparse, so then the value is in `options['emailaddress']`.

This is the equivalent command in Fjord:

https://github.com/mozilla/fjord/blob/master/fjord/base/management/commands/ihavepower.py

Comment (peterbe) [inline webapp-django/crashstats/authentication/management/commands/makesuperuser.py]: Oops. I think I hasted. I'll rectify.

Comment (peterbe) [inline webapp-django/crashstats/authentication/management/commands/makesuperuser.py]: https://github.com/mozilla/socorro/pull/3185

**Pull request merged**